src/routes/delegates/delegates.module.ts

```diff
@@ -1,9 +1,17 @@
 import { Module } from '@nestjs/common';
+import { GenericValidator } from '../../domain/schema/generic.validator';
+import { JsonSchemaService } from '../../domain/schema/json-schema.service';
+import { ValidationErrorFactory } from '../../domain/schema/validation-error-factory';
 import { DelegatesController } from './delegates.controller';
 import { DelegatesService } from './delegates.service';
 
 @Module({
   controllers: [DelegatesController],
-  providers: [DelegatesService],
+  providers: [
+    JsonSchemaService,
+    GenericValidator,
+    ValidationErrorFactory,
+    DelegatesService,
```

Comment:
Maybe it makes more sense to export these in the `DomainModule`. But since this is being used for the routes maybe we should create a new module that bundles: `JsonSchemaService`, `GenericValidator` and `ValidationErrorFactory`. What do you think?

Reply:
Yes, I think it's a good solution to this. What do you think about naming it `ValidationModule`? I've created it [04628c0](https://github.com/5afe/safe-client-gateway-nest/pull/271/commits/04628c01ebcec0eb0cd6129196bfb51ffdfc99c3). It's grouping the validation-related stuff, and located at the same level that `domain.module.ts`, do you think it makes sense?